e2e-tests/gef/cypress/integration/unissued-facilities-AIN.spec.js

```diff
@@ -0,0 +1,523 @@
+      applicationPreview.facilitySummaryListRowAction(2, 0).should('not.exist');
+      applicationPreview.facilitySummaryListRowAction(2, 1).should('not.exist');
+      applicationPreview.facilitySummaryListRowValue(2, 2).contains('Issued');
+      applicationPreview.facilitySummaryListRowAction(2, 2).should('not.exist');
+      applicationPreview.facilitySummaryListRowKey(2, 3).should('not.have.value', 'Date issued to exporter');
+      applicationPreview.facilitySummaryListRowValue(2, 3).contains('Date you submit the notice');
+      applicationPreview.facilitySummaryListRowAction(2, 3).should('not.exist');
+    });
+
+    it('clicking change should take you to about facility page with different url', () => {
+      const issuedDate = format(oneMonth, 'd MMMM yyyy');
+      const coverStart = format(twoMonths, 'd MMMM yyyy');
+      // should be able to change number 1 as changed to issued
+      applicationPreview.facilitySummaryListRowValue(3, 0).contains(MOCK_FACILITY_ONE.name);
+      applicationPreview.facilitySummaryListRowAction(3, 0).contains('Change');
+      applicationPreview.facilitySummaryListRowValue(3, 3).contains(coverStart);
+      applicationPreview.facilitySummaryListRowAction(3, 0).click();
+      cy.url().should('eq', relative(`/gef/application-details/${dealId}/unissued-facilities-change/${facilityOneId}/about-facility`));
+      aboutFacilityUnissued.facilityName().clear();
+      aboutFacilityUnissued.facilityName().type(`${MOCK_FACILITY_ONE.name}name`);
+      aboutFacilityUnissued.shouldCoverStartOnSubmissionYes().click();
+      aboutFacilityUnissued.cancelLink().click();
+      applicationPreview.facilitySummaryListRowValue(3, 0).contains(MOCK_FACILITY_ONE.name);
+      applicationPreview.facilitySummaryListRowAction(3, 0).contains('Change');
+      applicationPreview.facilitySummaryListRowAction(3, 0).click();
+      aboutFacilityUnissued.facilityName().clear();
+      aboutFacilityUnissued.facilityName().type(`${MOCK_FACILITY_ONE.name}name`);
+      aboutFacilityUnissued.shouldCoverStartOnSubmissionYes().click();
+      aboutFacilityUnissued.continueButton().click();
+      applicationPreview.facilitySummaryListRowValue(3, 0).contains(`${MOCK_FACILITY_ONE.name}name`);
+      applicationPreview.facilitySummaryListRowValue(3, 3).contains(issuedDate);
+    });
+
+    it('pressing submit button takes you to submit page and with correct panel once submitted to checker', () => {
+      applicationPreview.submitButtonPostApproval().click();
+      applicationSubmission.submissionText().contains('Someone at your bank must check your update before they can submit it to UKEF');
+      applicationSubmission.submitButton().click();
+      applicationSubmission.confirmationPanelTitleFacilities().contains('Issued facilities submitted for checking at your bank');
+    });
+  });
+});
+
+context('Unissued Facilities AIN - change to issued from preview page', () => {
```

Comment:
Is the first context for "all" facilities being changed? And this one is just for 1 or two changing?

I think this should go into it's own .spec file, at this point we're already at 300+ lines

Reply:
done - split into 2 files